cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -78,6 +80,8 @@ if(APPLE)
       bootstrap_cmd
       . "@HUNTER_GLOBAL_SCRIPT_DIR@/clear-all.sh" && ${bootstrap_cmd}
   )
+elseif(compiler_is_clang AND NOT CMAKE_CROSSCOMPILING)
+  list(APPEND bootstrap_cmd "--with-toolset=clang")
```

Comment:
I think same trick needed in `url_sha1_boost_library.cmake.in` too. Instructions I have used for testing: https://github.com/hunter-packages/docker-ubuntu-clang-only

Reply:
And for the `url_sha1_boost_ios_library.cmake.in` as well then

Reply:
I guess since everything already worked on OSX before and we don't expect someone with a weird setup were the boostrap script from Boost would fail to detect the correct toolset on OSX, I don't need to touch the iOS specific scheme then.